### DISCUSSION TITLE
Optimizing dbt Models with Redshift

### DISCUSSION THREAD

**Discussion by christineberger** [dbt-labs/devhub-writing#59] [Analytics craft]
Redshift's ability to tweak how you can sort and distribute data across nodes gives Redshift users a unique position to tune up their transformation flows with dbt. When following some best practices, we can effectively reduce run times (the best one I've done is from 7+ hours to ~3 hours just with some model refactoring, sort, and distribution strategies!)

Sort and distribution can sound scary, and to be honest, many Redshift users _know_ about them but up until implementing dbt, it was hard to take the time to figure out how to implement them in a meaningful way. With dbt, that's changed -  dbt makes it easy to see how your data relates and the insight you get from this is advantageous when mapping out your sort and dist configs. 

With this write up I will explain what each configuration means and what benefits they have to the database, as well as how you should architect your dbt models to get the best out of these configurations.

**Comment by amychen1776** [comment]
@christineberger  We are about to write a Redshift + dbt best practices whitepaper with Redshift. Would you be interested in joining in on the fun?

**Reply by christineberger** [reply]
@amychen1776 Absolutely!

**Comment by krevitt** [comment]
Hey @christineberger this would definitely be a great post!  Let's take this convo over to an issue where we + @amychen1776 can coordinate

**Reply by gr8khan** [reply]
Was this blog published? if so please provide link. Thanks!

**Reply by krevitt** [reply]
Not yet!  You can [follow this issue](https://github.com/dbt-labs/docs.getdbt.com/issues/1239), will be released in not too long
